src/ui/overview.rs

```diff
@@ -59,12 +59,16 @@ fn draw_logo_block<B: Backend>(f: &mut Frame<B>, app: &mut App, area: Rect) {
   text.patch_style(style_logo());
 
   // Contains the banner
-  let paragraph = Paragraph::new(text).block(Block::default().borders(Borders::ALL));
+  let paragraph = Paragraph::new(text).block(
+    Block::default()
+      .borders(Borders::ALL)
+      .border_type(BorderType::Rounded),
```

Comment:
I tried rounded border originally, but its renders weirdly on taller terminals and sometimes looks quite ugly and warped. I'm also pretty sure it will do weird things in windows. So I would prefer to keep it simple and use default borders

Reply:
What do you mean by tall terminal?
But yes, I never tried it on Windows :/

Reply:
I personally fins sharp corners nicer, but my main worry is this causing weirdness in some terminal/os config, especially powershell/cmd

Reply:
I tried on Windows PowerShell and Command Prompt, on Windows 11, and works fine on both.
But if you prefer sharp corners, then I'll revert them back :)

(ignore the errors since I have nothing there)

![Screenshot from 2022-01-24 20-10-07](https://user-images.githubusercontent.com/45714/150848282-e04dd456-939f-4bb4-940e-218d2dfc0b08.png)

Reply:
sharp again :)